### ISSUE TITLE
Migration in plugin is causing issues

### ISSUE THREAD

**Issue by clem21** [BitBagCommerce/SyliusCmsPlugin#497]
Hi,
We have been using your plugin on one of our live project for over a year now and we now want to update our project to Sylius 1.12.16 and have to update the CMS plugin to the 4.0.9 version. The problem is that in this new version the migration is directly including in the plugin causing issues on our database.
Do you have any idea on how to workaround this issue?

**Comment by stefandoorn** [comment]
I've manually registered them 'as executed' through the Symfony CLI commands.. but yeah, the migrations here should just detect if certain tables are already present and then silently finish execution.

**Comment by rust-le** [comment]
If you're managing migrations in the main project's namespace or if you have customized migrations already, you can disable auto-loading the plugin's migrations and replicate the necessary changes manually. If the migration structure from the plugin version conflicts with your existing schema, you can migrate your database incrementally instead of applying the full new version directly.

**Comment by stefandoorn** [comment]
Yes, we can - I mainly commented here that there was no guidance on this in an UPGRADE guide e.g.
